src/metabase/driver/sql_jdbc/sync/describe_database.clj

```diff
@@ -115,8 +115,8 @@
                           :type        (.getString rset "TABLE_TYPE")}))))))
 
 (defn- schema+table-with-select-privileges
-  [driver database]
-  (->> (driver/current-user-table-privileges driver database)
+  [driver conn]
+  (->> (sql-jdbc.sync.interface/current-user-table-privileges driver conn)
```

Comment:
I'm wondering whether we should call this with kwargs, so implementations can choose whether or not to support filtering by specific privileges.
```suggestion
  (->> (sql-jdbc.sync.interface/current-user-table-privileges driver conn :privileges #{:select}})
```

Reply:
Inclined to YAGNI on this, even if we decide to make this extension we might change our mind about the structure of the kwargs to use.

Reply:
Fair point, let's leave it